cmd/minikube/cmd/mount.go

```diff
@@ -36,42 +38,52 @@ var mountCmd = &cobra.Command{
 	Long:  `Mounts the specified directory into minikube.`,
 	Run: func(cmd *cobra.Command, args []string) {
 		if len(args) != 1 {
-			errText := `Please specify a driver name and a directory to be mounted:
-\tminikube mount MOUNT_DIRECTORY(ex:"/home")
+			errText := `Please specify the directory to be mounted: 
```

Comment:
The errors are different and there are unique/more-verbose errors for each incorrect input